test/UniV3LoanRouter.ts

```diff
@@ -391,7 +392,492 @@ describe("Uniswap V3 Loan Router", () => {
 
       const receipt = await tx.wait();
       const gasUsed = receipt.gasUsed;
-      expect(gasUsed.toString()).to.equal("477943");
+      expect(gasUsed.toString()).to.equal("477965");
+    });
+  });
+});
+
+interface WrapperTestContext {
+  router: UniV3LoanRouter;
+  mockCollateralToken: MockERC20;
+  mockWrapperToken: MockButtonWrapper;
+  bondFactory: BondFactory;
+  mockCashToken: MockERC20;
+  bond: BondController;
+  tranches: Tranche[];
+  user: Signer;
+  other: Signer;
+  signers: Signer[];
+}
+
+describe("Uniswap V3 Loan Router with wrapper", () => {
+  /**
+   * Sets up a test context, deploying new contracts and returning them for use in a test
+   */
+  const fixture = async (): Promise<WrapperTestContext> => {
+    const signers: Signer[] = await hre.ethers.getSigners();
+    const [user, other, admin] = signers;
+
+    const mockSwapRouter = <MockSwapRouter>await deploy("MockSwapRouter", signers[0], []);
```

Comment:
Why not use the destructured variable user on line 419?

Reply:
tru